src/Microsoft.DotNet.ExecutablePackageObtainer/Microsoft.DotNet.ExecutablePackageObtainer.csproj

```diff
@@ -0,0 +1,18 @@
+﻿<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <Version>$(SdkVersion)</Version>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <WarningsAsErrors>true</WarningsAsErrors>
+    <AssemblyOriginatorKeyFile>../../tools/Key.snk</AssemblyOriginatorKeyFile>
+    <SignAssembly>true</SignAssembly>
+    <PublicSign Condition=" '$(OS)' != 'Windows_NT' ">true</PublicSign>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="NuGet.Common" Version="$(CLI_NuGet_Version)" />
+    <PackageReference Include="System.Xml.XmlSerializer" Version="4.3.0" />
+  </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\Microsoft.DotNet.InternalAbstractions\Microsoft.DotNet.InternalAbstractions.csproj" />
+    <ProjectReference Include="..\Microsoft.DotNet.Cli.Utils\Microsoft.DotNet.Cli.Utils.csproj" />
+  </ItemGroup>
+</Project>
```

Comment:
Why are we creating a new assembly for this?

Reply:
yes, will it cause problem? I can merge it into dotnet.dll, however it is already pretty big

Reply:
or merge it to dotnet.dll but keep the name space separation?

Reply:
I think namespace and folder separation is enough. There's non-trivial overhead to having lots of tiny assemblies, not just at runtime, but also build/test time, + 13 more satellites, etc.

Reply:
i see, I'll merge it into dotnet.dll and use namespace and folder separation

Reply:
Fixed